### DISCUSSION TITLE
CI_JOB_NUMBER - is it used?

### DISCUSSION THREAD

**Discussion by mrchief** [ArtiomTr/jest-coverage-report-action#85] [Q&A] [answered]
I see it mentioned in the readme and it's set to a hardcoded value of 1 but a code scan shows this is not used anywhere. So, is this needed and if so, what's the intended use? There are a few ids available in the github context `github.run_number` and `github.run_id` so can they be used to set this value?

**Comment by ArtiomTr** [comment, marked as answer]
Hello @mrchief,

`CI_JOB_NUMBER` is not used anywhere. I will update configuration example in `README.md` file.

**Reply by ArtiomTr** [reply]
Removed via #93

**Reply by mrchief** [reply]
Awesome! Thanks for the quick response!
